src/objects/Hierarchy_Settings__c.object

```diff
@@ -355,6 +355,96 @@
         <type>Text</type>
         <unique>false</unique>
     </fields>
+    <fields>
+        <fullName>Prevent_Account_Deletion__c</fullName>
+        <defaultValue>false</defaultValue>
+        <description>Prevent an Account from being deleted if it has any related records (for example, Affiliation, Program Enrollment, or Course Connection records).</description>
```

Comment:
@kdy618 @lifewithryan @xinnanlisf  Is this ready for QA or do we need to make changes here?

(Did we decide earlier on "Reserved for future use."? I'm not sure what we ultimately decided!)

Reply:
@robertbwatson , yep we will use "Reserved for future use." This is now back to dev review. :)